flybywiresim/a32nx
# fix(mcdu): fuelpred running when inputting weights after engine start

Fixes #[issue_no]

## Summary of Changes
Currently the Fuel Prediction is not run when forgetting to put in Weights and CG before Engine start. 
This happens because `mcdu._fuelPredDone = true;` is never executed. 

This PR simply runs the same function that the INIT B page executes to set this value.

I am not sure if this is the cleanest way to resolve this, but this is only a quick fix and can be changed at a later date.
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub): Chaoz#3065

## Testing instructions

1. Start C/D
2. Perform normal setup without inserting Weights and CG and/or Blockfuel on the INIT B page
3. Start any Engine
4. INIT B page should switch to FUEL PRED with everything dashed except for FOB
5. Insert weights 
6. After a delay the page should populate with the prediction results


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
